x-pack/plugins/ingest_manager/common/types/rest_spec/epm.ts

```diff
@@ -27,15 +27,7 @@ export interface GetPackagesResponse {
     Installable<
       Pick<
         RegistryPackage,
-        | 'name'
-        | 'title'
-        | 'version'
-        | 'description'
-        | 'type'
-        | 'icons'
-        | 'internal'
-        | 'download'
-        | 'path'
+        'name' | 'title' | 'version' | 'description' | 'type' | 'icons' | 'download' | 'path'
```

Comment:
Not required for anything in this PR but found this while working on it. EPM doesn't return the `internal` key